src/bidiMapper/modules/context/BrowsingContextImpl.ts

```diff
@@ -1226,10 +1227,105 @@ export class BrowsingContextImpl {
             ...startNodes,
           ],
         };
-      case 'accessibility':
-        throw new UnsupportedOperationException(
-          'accessibility locator is not supported yet'
+      case 'accessibility': {
+        // https://w3c.github.io/webdriver-bidi/#locate-nodes-using-accessibility-attributes
+        if (!locator.value.name && !locator.value.role) {
+          throw new InvalidSelectorException(
+            'Either name or role has to be specified'
+          );
+        }
+        const bindings = await realm.evaluate(
+          /* expression=*/ '({getAccessibleName, getAccessibleRole})',
+          /* awaitPromise=*/ false,
+          /* resultOwnership=*/ Script.ResultOwnership.Root,
+          /* serializationOptions= */ undefined,
+          /* userActivation=*/ false,
+          /* includeCommandLineApi=*/ true
         );
+
+        if (bindings.type !== 'success') {
+          throw new Error('Could not get bindings');
+        }
+
+        if (bindings.result.type !== 'object') {
+          throw new Error('Could not get bindings');
+        }
+        return {
+          functionDeclaration: String(
+            (
+              name: string,
+              role: string,
+              bindings: any,
+              maxNodeCount: number,
+              ...startNodes: Element[]
+            ) => {
+              const returnedNodes: Element[] = [];
+
+              function collect(
+                contextNodes: Element[],
+                selector: {role: string; name: string}
+              ) {
+                for (const contextNode of contextNodes) {
+                  let match = true;
+
+                  if (selector.role) {
+                    const role = bindings.getAccessibleRole(contextNode);
+                    if (selector.role !== role) {
+                      match = false;
+                    }
+                  }
+
+                  if (selector.name) {
+                    const name = bindings.getAccessibleName(contextNode);
+                    if (selector.name !== name) {
+                      match = false;
+                    }
+                  }
+
+                  if (match) {
+                    if (
+                      maxNodeCount !== 0 &&
+                      returnedNodes.length === maxNodeCount
+                    ) {
+                      break;
+                    }
+
+                    returnedNodes.push(contextNode);
```

Comment:
I guess we don't need to make another loop if the `maxNodeCount` is reached.
```suggestion
                    returnedNodes.push(contextNode);

                                        if (
                      maxNodeCount !== 0 &&
                      returnedNodes.length === maxNodeCount
                    ) {
                      break;
                    }
```

Reply:
I agree, but note that this is not per spec

Reply:
wait, it's actually wrong, we need to check before we append.

Reply:
oh, right. So you have to `return` instead of `break`. Otherwise it will have lots of unnecessary DOM traversing.

Reply:
Return would not be sufficient too, because it is in the recursive function

Reply:
added an abort flag